cmd/managementd/main.go

```diff
@@ -62,6 +63,7 @@ func main() {
 	router.HandleFunc("/3G-connectivity", managementinterface.ThreeGConnectivityHandler).Methods("GET")
 	router.HandleFunc("/API-server", managementinterface.APIServerHandler).Methods("GET")
 	router.HandleFunc("/network-interfaces", managementinterface.NetworkInterfacesHandler).Methods("GET")
+	router.HandleFunc("/interface-status/{id:[0-9]+}", managementinterface.CheckInterfaceHandler).Methods("POST")
```

Comment:
I don't know sorry.  I got my nephew to write this network interfaces code because I'm so new to web programming I wasn't sure how to do it.

If we change that line there to GET, will that mean changes to the network-interfaces.html file will need to be changed, specifically the javascript call to Golang there?

Reply:
This should definitely be a GET.

It should be easy to change - just this line here and the JavaScript which calls it.